docs/topic-guides/training-units.txt

```diff
@@ -0,0 +1,180 @@
+.. _topic-guides_training-units:
+
+################
+ Training Units
+################
+
+With the flexibility of Determined’s experiment configuration, the combinations of possible
+configuration settings available are numerous. This guide aims to help you better understand the
+options available for training units to make the right choice for your model.
+
+Determined supports configuring the following fields:
+
+-  ``searcher.max_length`` in most searchers
+
+-  ``searcher.length_per_round`` in PBT
+
+-  ``min_validation_period``
+
+-  ``min_checkpoint_period``
+
+-  ``searcher.budget`` when using ``adaptive_simple``
+
+in the following units:
+
+-  ``records``: A *record* is a single labeled example (sometimes called a sample).
+
+-  ``batches``: A *batch* is a group of records (the number of records in a batch is configured via the ``global_batch_size`` hyperparameter).
+
+-  ``epochs``: An *epoch* is typically single copy of the entire training data set.
+
+
+The sections below present different configurations that result in the same effective training
+length and behavior but use different training units.
+
+In general, conversions like the conversions between the configurations below will result
```

Comment:
"conversions like the conversions between the configurations below will result in identical behavior" -> don't follow this.

Reply:
Yeah, cleaned this up. Should be much better now.